docker-compose.yaml

```diff
@@ -10,6 +10,8 @@ services:
         grafana_version: ${GRAFANA_VERSION:-10.2.0}
     ports:
       - 3000:3000/tcp
+    environment:
+      GF_LOG_LEVEL: 'debug'
```

Comment:
Always on? Also in CI pipeline (e.g.)?

Reply:
We could change it to something like `GF_LOG_LEVEL: ${LOG_LEVEL}`, but I do honestly not see a problem enabling debug mode in the pipeline as well

Reply:
I think it's very chatty